tests/suites/test_suite_psa_crypto.function

```diff
@@ -3152,6 +3480,1029 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void aead_multipart_encrypt( int key_type_arg, data_t *key_data,
+                             int alg_arg,
+                             data_t *nonce,
+                             data_t *additional_data,
+                             int do_test_ad_chunked,
+                             data_t *input_data,
+                             int do_test_data_chunked,
+                             int do_set_lengths,
+                             data_t *expected_output )
+{
+    size_t ad_part_len = 0;
+    size_t data_part_len = 0;
+
+    TEST_ASSERT( do_test_ad_chunked || do_test_data_chunked );
```

Comment:
Please explain this check in a comment. It's not that the request wouldn't make sense or that the test wouldn't work, but that the test would do nothing and therefore this would be very likely to be a mistake in the test data.